zds/tutorialv2/publication_utils.py

```diff
@@ -276,6 +279,28 @@ def publish(self, md_file_path, base_name, change_dir=".", pandoc_debug_str="",
             self.__logger.info("Finished {} generation".format(base_name + "." + self.format))
 
 
+class NumberOfWordPublicator(Publicator):
+    """
+    Compute the number of words for a given content
+    """
+    def __init__(self):
+        pass
+
+    def publish(self, md_file_path, base_name, silently_pass=True, **kwargs):
```

Comment:
Cette méthode est juste une surcharge de Publicator, si je ne laisse pas le
nom, ça marche pas.

2016-06-23 14:17 GMT+02:00 Gérard Paligot <notifications@github.com>:

> In zds/tutorialv2/publication_utils.py
> <https://github.com/zestedesavoir/zds-site/pull/3681#discussion_r68223706>
> :
>
> > @@ -276,6 +279,28 @@ def publish(self, md_file_path, base_name, change_dir=".", pandoc_debug_str="",
> >              self.__logger.info("Finished {} generation".format(base_name + "." + self.format))
> >
> >
> > +class NumberOfWordPublicator(Publicator):
> > +    """
> > +    Compute the number of words for a given content
> > +    """
> > +    def __init__(self):
> > +        pass
> > +
> > +    def publish(self, md_file_path, base_name, silently_pass=True, **kwargs):
>
> Cette méthode est très mal nommée. Dans le script de migration, j'ai cru
> que tu comptais re-publier tous les contenus !
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3681/files/82f4ad010f235ffc4d62163669ce8dd55a955340#r68223706>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABc_xRMQvzpHbpmDq1oShBPXeIndXyXgks5qOnlTgaJpZM4I7sFO>
> .
>